.github/workflows/ci_action_noetic.yml

```diff
@@ -7,50 +7,47 @@ env:
   CATKIN_LINT: true
   CATKIN_LINT_ARGS: '--strict'
   CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
+  ROS_DISTRO: noetic
 
 jobs:
-  build_test_main:
-    name: "Build + Test with Main Repo of melodic (http://packages.ros.org/ros/ubuntu)"
-    runs-on: ubuntu-18.04
+  testing-repo:
+    name: "Build + Test with Testing Repo of noetic (http://packages.ros.org/ros-testing/ubuntu)"
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v1
       - uses: 'ros-industrial/industrial_ci@master'
         env:
-          ROS_REPO: main
-          ROS_DISTRO: melodic
-    continue-on-error: true
+          ROS_REPO: testing
 
-  build_test_testing:
-    name: "Build + Test with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
-    runs-on: ubuntu-18.04
+  main-repo:
+    name: "Build + Test with Main Repo of noetic (http://packages.ros.org/ros/ubuntu)"
+    runs-on: ubuntu-20.04
+    continue-on-error: true
     steps:
       - uses: actions/checkout@v1
       - uses: 'ros-industrial/industrial_ci@master'
         env:
-          ROS_REPO: testing
-          ROS_DISTRO: melodic
+          ROS_REPO: main
 
   clang-format:
-    name: "Build + Test with clang format with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
-    runs-on: ubuntu-18.04
+    name: "Build + Test with clang format with Testing Repo of noetic (http://packages.ros.org/ros-testing/ubuntu)"
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v1
       - uses: 'ros-industrial/industrial_ci@master'
         env:
           ROS_REPO: testing
-          ROS_DISTRO: melodic
           CLANG_FORMAT_CHECK: file
-          CLANG_FORMAT_VERSION: 3.9
+          CLANG_FORMAT_VERSION: 10
```

Comment:
What is the reason behind this?

Reply:
I don't know, why it was `3.9` before. Do you know the reason?

Anyways, now the version corresponds to the default on ubuntu focal.